overlord/snapstate/handlers.go

```diff
@@ -737,6 +738,31 @@ func (m *SnapManager) undoMountSnap(t *state.Task, _ *tomb.Tomb) error {
 	return m.backend.RemoveSnapDir(snapsup.placeInfo(), otherInstances)
 }
 
+// saveSnapDisabledServices queries for all currently disabled snap services and
+// adds those inside *snap.Info.
```

Comment:
the comment is wrong, should say SnapState ?

Reply:
yes, but I changed the comment for the new querySnapDisabledServices anyways so this has gone away